can_tools/scrapers/official/TX/texas_vaccine.py

```diff
@@ -0,0 +1,113 @@
+import pandas as pd
+import us
+import lxml.html
+import requests
+from can_tools.scrapers import CMU
+from can_tools.scrapers.official.base import StateDashboard
+
+# the crename keys became absurdly long so I store them in another file
+# if there's a way to 'find and replace' values inside the CMU objects, that would cut the length to 1/3 the current
+from .tx_vaccine_crenames import crename, crename_female, crename_male, crename_unknown
+
+
+class TexasVaccine(StateDashboard):
+    has_location = False
+    state_fips = us.states.lookup("Texas").fips
+    location_type = "county"
+    source = "https://www.dshs.state.tx.us/coronavirus/immunize/vaccine.aspx"
+
+    def fetch(self) -> requests.models.Response:
+        fetch_url = "https://www.dshs.state.tx.us/immunize/covid19/COVID-19-Vaccine-Data-by-County.xls"
+        res = requests.get(fetch_url)
+        if not res.ok:
+            raise ValueError("Could not fetch download file")
+        return res
+
+    def normalize(self, data) -> pd.DataFrame:
+        data = pd.ExcelFile(data.content)
+        df = data.parse("By County").rename(columns={"County Name": "location_name"})
+        df["dt"] = self._retrieve_dt("US/Eastern") - pd.Timedelta(days=1)
```

Comment:
This is just like the snippet I recommended for DC :)

This seems useful. What do you think about adding a `_retrieve_dtm1d` (datetime minus 1 day) to the base class (found in `can-scrapers/can_tools/scrapers/base.py`) similar to `_reterieve_dt`?